apps/src/code-studio/progressRedux.js

```diff
@@ -604,7 +593,7 @@ const levelWithStatus = (
   return {
     ...normalizedLevel,
     status: statusForLevel(level, levelResults),
-    isCurrentLevel: isCurrentLevel(currentLevelId, normalizedLevel),
```

Comment:
Do we need to cover this with a test?

Reply:
i feel like [this test](https://github.com/code-dot-org/code-dot-org/blob/68f62632745f99ace7e0e29a65d77fb0efc4b7bd/apps/test/unit/code-studio/progressReduxTest.js#L829) is probably good enough. i could update it to have the other level ids include `currentLevelId` in the string, but that seems like a bizarre and potentially confusing thing to test with the new implementation. what do you think?

Reply:
You're right, that test seems to cover it, no need to add another

Reply:
👍